### DISCUSSION TITLE
docker compose V2 default container name

### DISCUSSION THREAD

**Discussion by Tardo** [Tecnativa/doodba-copier-template#389] [General]
As far as I have been able to see the "new" version of docker compose changes the way containers are named by default.

Until now, the name was something like: ```<project_name>_<service>_<counter>```.

With the new version: ```<project_name>-<service>-<counter>```

This is going to cause problems in some CI/CD. Since it will now affect all of us, it is still a good idea to use "container_name" to give them a predictable name.

What do you think? I don't know if you can configure docker to set the default container name to keep ```<project_name>_<service>_<counter>```.

**Comment by pedrobaeza** [comment]
Which problem do you predict?

**Reply by Tardo** [reply]
Trying to invoke a task such as launching a backup to the ```<foo>_backup_1``` container and it fails to find it after starting using "docker compose" and not "docker-compose".

This happens when using "docker exec" (you need to specify container name). Those who use "docker compose exec" (you need to specify project and service) will not have this problem.

**Reply by pedrobaeza** [reply]
OK, I see. Not sure if that can be fixed through the YAML file. It's worth to check it.
